07-workbase/04-schema-designer.md

```diff
@@ -0,0 +1,183 @@
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_attribute_6.png)[body:end]
+[footer:start] [footer:end]
+[slide:end]
+
+</div>
+
+### Define New Relationship Type
+<div class="slideshow">
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_relationship_1.png)[body:end]
+[footer:start]We can use the left bar to deine new relationship type[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_relationship_2.png)[body:end]
+[footer:start]We can specify the name of the relationship and the role players[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_relationship_3.png)[body:end]
+[footer:start]We can specifiy the attributes that the relationship can have, given that the attribute type has already been defined [footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_relationship_4.png)[body:end]
+[footer:start]We can specify the roles the relationship type can play, given that the relationship for those roles have been defined [footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_relationship_5.png)[body:end]
+[footer:start] Create the type by clikcing on __Submit__ [footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_relationship_6.png)[body:end]
+[footer:start] [footer:end]
+[slide:end]
+
+</div>
+
+### Manage Attribute Types
+<div class="slideshow">
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/attribute_type_1.png)[body:end]
+[footer:start]We may add an exisiting attribute type to a schema concept type by selecting it and clicking on __Add Attribute Types__ in the right bar.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/attribute_type_2.png)[body:end]
+[footer:start]Select the attribute we want to add.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/attribute_type_3.png)[body:end]
+[footer:start]Click __Add__. [footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/attribute_type_4.png)[body:end]
+[footer:start] [footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/attribute_type_5.png)[body:end]
+[footer:start]We can remove an attribute by clicking on the trash icon next to the attribtue. [footer:end]
+[slide:end]
+</div>
+
+### Display Settings
+<div class="slideshow">
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/schema_display_settings_1.png)[body:end]
+[footer:start]Navigating to the display settings by clicking on the __gear__ icon in the right bar.[footer:end]
```

Comment:
Navigate~~ing~~ to the **Display Settings** by clicking on the **gear** icon ~~in the right bar~~.

Given the way we fade screenshots, we don't need to describe the location when we're sure it's obvious. Please do take note of this in other footer notes.

Reply:
**Display Settings** (initial letters are capital)